### PR TITLE
initial functions

### PR DESCRIPTION
**Pull request by ujjawal-khare-27**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module example.com/m/v2
```
Comment (abhijeetgurle) [inline go.mod]: We should add github path of this package in front of module:-

Reference:- https://go.dev/doc/tutorial/getting-started#code

**Review by abhijeetgurle — CHANGES_REQUESTED** [review]

File: main.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	}
+
+	ds.documents[document.ID] = &document
+	return nil
+}
+
+// SaveToFile -> takes fileName -> Save documentStore to file with given filename
+// LoadFromFile -> filename -> Load document to file with given filename
+
+func (ds *DocumentStore) SaveToFile(fileName string) error {
```
Comment (abhijeetgurle) [inline main.go:55]: @ujjawal-khare-27 I think you should use the functions which I am writing to write & read from files.

Comment (abhijeetgurle) [reply main.go:55]: @ujjawal-khare

Comment (ujjawal-khare-27) [reply main.go:55]: Checking, will do today

**Review by abhijeetgurle — APPROVED** [review]

**Pull request merged**